rules/cloud/azure/azure_ad_suspicious_signin_bypassing_mfa.yml

```diff
@@ -0,0 +1,28 @@
+title: Potential MFA Bypass Using Legacy Client Authentication
+id: 53bb4f7f-48a8-4475-ac30-5a82ddfdf6fc
+status: experimental
+description: Detect successful authentication from potential clients using legacy authentication via UserAgent strings. This could be a sign of MFA bypass using a password spray attack.
```

Comment:
```suggestion
description: Detects successful authentication from potential clients using legacy authentication via user agent strings. This could be a sign of MFA bypass using a password spray attack.
```